src/libraries/System.Private.CoreLib/src/System/Text/ASCIIUtility.cs

```diff
@@ -42,6 +42,28 @@ private static bool AllCharsInUInt64AreAscii(ulong value)
             return (value & ~0x007F007F_007F007Ful) == 0;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static int GetIndexOfFirstNonAsciiByteInLane(Vector128<byte> value, Vector128<byte> bitmask)
```

Comment:
>InLane [](start = 54, length = 6)

minor nit: There are 16 lanes in `Vector128<byte>` and you are trying to find first lane that has non-ascii byte. So probably rename it to `GetIndexOfFirstNonAsciiByte_Arm64` or something.